code/modules/client/preference_setup/global/preferences.dm

```diff
@@ -333,3 +333,12 @@ var/global/list/_client_preferences_by_type
 	options = list(PREF_SHOW, PREF_HIDE)
 	default_value = PREF_HIDE
 	flags = R_ADMIN|R_DEBUG
+
+/********************
+* Area Info Blurb *
+********************/
+
+/datum/client_preference/area_info_blurb
+	description ="Show area information"
+	key = "AREA_INFO"
+	default_value = PREF_YES
```

Comment:
Honestly I would prefer this to default to PREF_NO, though I'm not strongly opinionated either way.

Reply:
I have a feeling this will never be seen if it defaults to No, tbh.